.travis.yml

```diff
@@ -1,20 +1,33 @@
 dist: trusty
 language: go
-os:
-  - windows
```

Comment:
Sorry maybe I wasn't clear. We still need to run all but the file system  tests for Windows.

For example maybe put all unix OS specific tests in  `block_unix_test.go`

Reply:
> Sorry maybe I wasn't clear. We still need to run all but the file system tests for Windows.
> 
> For example maybe put all unix OS specific tests in `block_unix_test.go`

Currently, go fuse library do not support windows platform, so there is a compile error in windows platform
```
..\..\..\..\pkg\mod\github.com\hanwen\go-fuse@v1.0.0\fuse\types.go:511:2: undefined: Attr
```

I am trying to solve it and do you have any suggestions?
Thanks

Reply:
why do you think the library  is loaded when running the tests under windows?
When you have something like `block_unix_test.go` the file should be loaded only on unix OS-es

Reply:
I think you should rename all fuse utils files to fuse_..._unix.go

Reply:
> fuse_..._unix.go

For linux and macos, i add the build flag in golang files
```
// +build linux darwin
```

But for compiling it in windows platform, so there is fuse_xxx_windows.go.
The advantage  is that other developers can write fuse cases in everywhere and do not need to put fuse cases in a special directory.

From the build flag, there is only linux、darwin、windows、freebsd suffix to support different platform. I am not sure the files which suffix is unix used in linux and macos.

Reply:
Good point, but looking at the code seems that these tests will fail under windows anyway as they will not use the fuse server mounts so  can't inject errors and when the tests expects an error, no error will be returned.

Reply:
@krasi-georgiev 
Currently, for windows platform, it can all cases exclude fuse related cases and for linux and macosx plateform, it can rull all cases. They works well.

I think the tests which need to inject filesystem errors only run in linux and macosx platform.
And in windows platform, it only run other cases.

Do you think it is ok?